Update SwiSBox XML

d-spoerri-swistec opened the pull request:
As discussed via E-Mail with @ergo-furrer 

Changes:
- Change device category to SubMeterElectricity
- Fix register numbers
- Fix functional profile names, and texts
- Add apparent energy and reactive power
- Remove duplicate entry for apparent power

I've tested this XML (with the relay FPs removed) using the java library. Please take a close look at it, as I'm not aware of the specific requirements regarding these XMLs.

ArndGerns commented on XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml:
```diff
@@ -1390,6 +1402,7 @@
           </dataPointList>
         </functionalProfileListElement>
       </functionalProfileList>
+      </functionalProfileList>
```
This is a syntax error (double `</functionalProfileList>`).

ArndGerns commented on XMLInstances/ExtInterfaces/SGr_02_mmmm_dddd_SwiSBox_Actuator.xml:
```diff
@@ -426,253 +424,259 @@
                 </legibleDescription>
+                <legibleDescription>
+                  <textElement>Total reactive power measurement</textElement>
+                  <language>fr</language>
```
Why French?

ArndGerns commented:
At some places the text formatting was removed - in the following file, I have restored the formatting (renamed to `.txt` as it was not possible to add a `.xml`).

[SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.xml](https://github.com/SmartGridready/SGrSpecifications/files/13759827/SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.txt)

Furthermore, I have removed the relais for now. AFAIK this was discussed with Maike Schubert / Deklarationsstelle.

If the changed version is OK for you, we can check it in into the main repository. Then you need to discuss with the Deklarationsstelle to get it published.

d-spoerri-swistec commented:
Thanks for your feedback and corrections. It is correct, that the relays will not be part of the initial declaration, but I've re-added them as I was not sure if we should leave them in the XML (I must have also added the duplicate tag while doing that).
I've added all of your changes to my XML as well now, so from my view, a merge should be ok now.

ArndGerns reviewed (commented):
Hi @d-spoerri-swistec,

the content of the XML is good now. There are two things, that should be checked:

* File name should be `SGr_02_mmmm_dddd_SwiSBox_SubMeterElectricity.xml` as the category should be part of the file name
* The file has unix-style line endings with only NL - other files have windows-style line endings with CR+NL. Need to see whether this gives problems.

If you're OK, we can merge it, and I will do these two adaptions.

Best regards,
Arnd.

d-spoerri-swistec commented:
@ArndGerns thanks for your feedback and all the support, II don't think there's any more changes from our side, so this may be merged. Thanks!

LTA-WKP commented:
@ArndGerns: As discussed the IDs for the file names
UnternehmensID: 4893879785
DeklarationsID: 8288144069

BR
Luca

ArndGerns commented:
Hi @LTA-WKP, @wkp-maike,

I have renamed the EID XML accordingly, It should now be ready for declaration.

Best regards,
Arnd.
